website/splash_table.py

```diff
@@ -63,27 +72,22 @@ def make_table(results: list):
     return cds, dt
 
 
-def querying(qrystr: str = 'notanobject'):
+def querying(query_string: str = ''):
     """
     Queries the database for an object
 
     Parameters
     ----------
-    qrystr
+    query_string
         The partial string of an object
 
     Returns
     -------
     The list of tuples of object info
     """
-    qry = Qry(qrystr)
     db = load_db()  # load the database
-    results = db.search_object(qrystr)
-    if qry.default:
-        return results
-    else:
-        results = [obj[:3] for obj in results]
-        return results
+    results = db.search_object(query_string, format='pandas')
```

Comment:
using pandas dataframe instead of list for enhanced readability and general betterness